scripts/post-install

```diff
@@ -0,0 +1 @@
+su -s /bin/bash ktranslate -c "git clone https://github.com/kentik/snmp-profiles.git /etc/ktranslate/profiles"
```

Comment:
Could we just include this as part of the package instead of having to have a post script to pull?

Reply:
Great q -- I went around on this for a while and couldn't get it to work. Specifically, https://github.com/kentik/pkg doesn't support globs for which files to get put into a package. We could script the add of each file in the workflow, but that seemed a bit messy. 

Separately, pulling the snmp profiles out of here make versioning ktrans code a lot easier. No need for a new release every time a snmp profile gets changed. Con is that it pushes keeping up to date here onto users, but that seems reasonable. 

Typing this made me realize though that installing a new version of ktrans will error out because the git repo will already exist. Time for a more complex script which knows to git update in this case I think.

Reply:
Got it -- makes sense :+1: